### ISSUE TITLE
Each saving adds an empty p Tag

### ISSUE THREAD

**Issue by chequille** [benjaminkott/bootstrap_package#324]
Hi,
I am using Typo3 7.6.4 and the latest bootstrap_package from the repository.
I found, that each saving of a content element adds an adittional <p></p> in the Frontend Ouput.
Any idea why this happens?
Best regards,
Chequille

**Comment by Outdoorsman** [comment]
This is also happening for me. I have not idea where to starting looking for an answer or if it is a TYPO3 core issue... just know that it's cause a lot of trouble. It even happens when I change the RTE to code view and save.

**Comment by chequille** [comment]
Hi Outdoorsman,
I thought as well that RTE is doing it. But I think in this case it is not. I made a small extension for some own content elements using flux, fluid and vhs view helper. In this case, the p tags were not added, everything is fine.
Therefore I think only Benjamin can help :-)

**Comment by Outdoorsman** [comment]
Good feedback @chequille. I'd be curious if @benjaminkott or others can reproduce this or needs more info if he can't.

**Comment by thomasmstein** [comment]
Can confirm, same or similar problem. 
I use Typo3 7.6.4 with bootstrap_package 7.0.0 with the default htmlArea RTE. 
RTE added tags I did not want or thought not have selected? Sometime I could not work out what made the content element look like it did (even in code view). Spaces between content elements became bigger and bigger. RTE made the system unusable and extremely slow. Copy and paste content even from a normal editor worked with long time delays. I eventually had to removed RTE altogether (a pity) and everything works fine now ... back to old style hand coding :-)
